practice_app/pages/index.js

```diff
@@ -75,6 +80,22 @@ export default function Home({ context }) {
         }
     }
 
+    const getNumber = async () => {
+        if (countr == null) {
+            setNumb('Please select a location by clicking on the map')
+            return
+        }
+        const { data } = await axios.get(`api/getStatistics?country=${countr}`)
+
+        if (data.answered == 'yes') {
+            setNumb(
+                `Country: ${countr} Death Toll:  ${data.deathToll} Recovered People:  ${data.recovery} Infected People: ${data.infection}`,
+            )
+        } else {
+            setNumb('No information available about your country')
+        }
+    }
+
```

Comment:
1. Use a better name. This shouldn't be called `getNumber`, should be called something like `getCovidStats`
 2. The way you are doing it is fine, but a cleaner approach would be to `setCovidData({country, deathToll, recovery, infection})` and leave the "presentation formatting" when you are rendering.

Reply:
done the 1st one.